Source/Engine/UI/GUI/Common/TextBoxBase.cs

```diff
@@ -282,6 +282,111 @@ public bool IsSelectable
         [EditorDisplay("Border Style"), EditorOrder(2011), Tooltip("The color of the border when control is focused (Transparent if not used)")]
         public Color BorderSelectedColor { get; set; }
 
+        /// <summary>
+        /// The border rectangle. Used to draw border around the text box.
+        /// </summary>
+        [EditorDisplay("Border Style"), EditorOrder(2012), Tooltip("The border rectangle. Used to draw border around the text box.")]
+        public TextBoxBorder Border { get; set; }
+
+        /// <summary>
+        /// The background rectangle. Used to draw background of the text box.
+        /// </summary>
+        [Serializable]
+        public struct TextBoxBorder
+        {
+            public Color Color { get; set; } = Color.Transparent;
+            public Color SelectedColor { get; set; } = Color.Transparent;
+
+            /// <summary>
+            /// The border lines width. Use 0 to disable border.
+            /// </summary>
+            public float Width { get; set; }
```

Comment:
I'm unsure about this one, it seems like the other 4 width properties already store that information.